cpu/native/startup.c

```diff
@@ -455,6 +475,22 @@ __attribute__((constructor)) static void startup(int argc, char **argv, char **e
             case 'z':
                 _zep_params_setup(optarg, zeps++);
                 break;
+#endif
+#ifdef MODULE_PERIPH_SPIDEV_LINUX
+            case 'p': {
+                long bus = strtol(optarg, &optarg, 10);
+                if (*optarg != ':') {
+                    usage_exit(EXIT_FAILURE);
+                }
+                long cs = strtol(++optarg, &optarg, 10);
+                if (*optarg != ':') {
+                    usage_exit(EXIT_FAILURE);
+                }
+                if (spidev_linux_setup(bus, cs, ++optarg) < 0) {
+                    usage_exit(EXIT_FAILURE);
+                }
+                }
```

Comment:
This looks weird. Either go one indent back or put the whole block one indent in more.